lib/irb/context.rb

```diff
@@ -150,25 +150,14 @@ def initialize(irb, workspace = nil, input_method = nil)
       end
 
       @user_aliases = IRB.conf[:COMMAND_ALIASES].dup
-      @command_aliases = @user_aliases.merge(KEYWORD_ALIASES)
+      @command_aliases = @user_aliases.dup
```

Comment:
I think `@user_aliases` is not referenced anymore after this change:

```suggestion
      @command_aliases = IRB.conf[:COMMAND_ALIASES].dup
```